.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Main CI
+
+on: [push, pull_request]
+
+env:
+  DB_HOST: '127.0.0.1'
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Init Database
+        run: |
+          mysql -uroot -h127.0.0.1 -proot -e 'CREATE DATABASE IF NOT EXISTS `invo`;'
+
+      - name: Copy .env file
+        run: cp .env.example .env
+
+      - name: Add Phalcon package repository
+        run: curl -s https://packagecloud.io/install/repositories/phalcon/mainline/script.deb.sh | sudo bash
```

Comment:
Do we really need install Phalcon by hand?

Reply:
Copy/paste from migrations .yml. Need to change in both.